### DISCUSSION TITLE
XpensAuditor v1.0 11-26-22 Released

### DISCUSSION THREAD

**Discussion by kaushikjadhav01** [NCSU-Fall-2022-SE-Project-Team-11/XpensAuditor---Group-11#39] [Announcements]
https://github.com/NCSU-Fall-2022-SE-Project-Team-11/XpensAuditor---Group-11/releases/tag/1.0

## What's Changed
* Feature: Online E-commerce product searcher by @kaushikjadhav01 in https://github.com/Surya-06/XpensAuditor---Group-11/pull/13
* Feature: Phone Number verification by @kaushikjadhav01 in https://github.com/Surya-06/XpensAuditor---Group-11/pull/13
* Fix: Ratings were not being stored to Firebase by @kaushikjadhav01 in https://github.com/Surya-06/XpensAuditor---Group-11/pull/17
* Fix: Suggestions were not being stored to Firebase by @kaushikjadhav01 in https://github.com/Surya-06/XpensAuditor---Group-11/pull/17
* Feature: Alert users when overwriting Ratings & Suggestions by @kaushikjadhav01 in https://github.com/Surya-06/XpensAuditor---Group-11/pull/17
* Fix: Build and test pipeline start-up & execution by @Surya-06 in https://github.com/Surya-06/XpensAuditor---Group-11/pull/8
* Feature: Add email notification when there are shared users. by @Surya-06 in https://github.com/Surya-06/XpensAuditor---Group-11/pull/5
* Fix: gear icon navigation in side bar by @Surya-06 in https://github.com/Surya-06/XpensAuditor---Group-11/pull/10

**Full Changelog**: https://github.com/Surya-06/XpensAuditor---Group-11/compare/0.0.1...1.0

**Comment by Hp-Harshitha-hp** [comment]
This release caused issue#20
https://github.com/NCSU-Fall-2022-SE-Project-Team-11/XpensAuditor---Group-11/issues/20

**Comment by kaushikjadhav01** [comment]
Fixed Issue https://github.com/NCSU-Fall-2022-SE-Project-Team-11/XpensAuditor---Group-11/issues/20 in https://github.com/NCSU-Fall-2022-SE-Project-Team-11/XpensAuditor---Group-11/pull/13
